app/uk/gov/hmrc/tariffclassificationfrontend/models/Case.scala

```diff
@@ -36,5 +37,9 @@ case class Case
   keywords: Set[String] = Set.empty
 ) {
 
-  def rulingHasNotExpired: Boolean = decision.flatMap(_.effectiveEndDate).exists(_.compareTo(Instant.now) >= 0)
+  def rulingHasExpired(implicit appConfig: AppConfig): Boolean = {
```

Comment:
I think this would read better as `hasLiveRuling`

Reply:
I will change it to `hasExpiredRuling`